### DISCUSSION TITLE
[suggestion] additional menus on sound quality (bitlate) and rendering

### DISCUSSION THREAD

**Discussion by 97lily** [losnoco/Cog#349] [Ideas]
First of all, I would like to thank you for your ongoing development.

Now, to the suggestion made in the subject line, some music players, such as "Droidsound-e", have a setting that allows the player to be set to "linear" mode, which renders the output as more approximate than the actual output, and also has a setting to change the sample rate.
I initially thought that the developer had some reason for not having them as a setting in "cog", but is it possible to add a setting to output them as a clear sound source?

Finally, we use machine translation, so in case there is any context that is difficult to understand, you can reply to it :)

**Comment by kode54** [comment]
Please see: Synthesis.
